python/lsst/pipe/analysis/analysis.py

```diff
@@ -548,10 +548,13 @@ def plotAgainstMagAndHist(self, log, filename, stats=None, camera=None, ccdList=
 
         labelVisit(filename, plt, axScatter, 1.18, -0.11, color="green")
         if zpLabel is not None:
+            yOff = 0.02 if uberCalLabel is not None else 0
+            yOff = -0.02 if "_2" in zpLabel else yOff
```

Comment:
This logic is hard to follow.  I think it's worth splitting it apart into multiple explicit if/elif/else statements.

Reply:
I tried, but it actually didn't look much clearer.  I opted to leave it as is, but added the following explanatory comment:
>\# The following sets yOff to accommodate the longer labels for the
>\# compare scripts and/or for the presence of the extra uberCalLabel
>\# (for coadds).  Font size is adjusted with uberFontSize below.